crates/factor-outbound-http/src/intercept.rs

```diff
@@ -0,0 +1,107 @@
+use http::{Request, Response};
+use http_body_util::{BodyExt, Full};
+use spin_world::async_trait;
+use wasmtime_wasi_http::{body::HyperOutgoingBody, HttpResult};
+
+pub type HyperBody = HyperOutgoingBody;
```

Comment:
This seems a bit unnecessary? I don't see how `HyperBody` is clearer than `HyperOutgoingBody`